featuretools/selection/selection.py

```diff
@@ -170,16 +170,20 @@ def remove_highly_correlated_features(
         more_complex_name = columns_to_check[i]
         more_complex_col = fm_to_check[more_complex_name]
 
-        # Convert boolean column to be float64
-        if pd.api.types.is_bool_dtype(more_complex_col):
+        # Convert boolean or Int64 column to be float64
+        if pd.api.types.is_bool_dtype(more_complex_col) or isinstance(
```

Comment:
Changing `NumCharacters` to return `IntegerNullable` triggered an error here. Int64 doesn't seem to support the `corr` operation. There is a `_is_integer_dtype` function, but that would cast a wider net and grab series with the `int64` dtype. This way is a little more selective and does the conversion less.